client/tests/integration/multiple_blocks_created.rs

```diff
@@ -15,10 +15,21 @@ const N_BLOCKS: usize = 510;
 #[test]
 fn long_multiple_blocks_created() {
     // Given
-    let (_rt, network, iroha_client) = <Network>::start_test_with_runtime(4, 1, Some(10_965));
+    let (_rt, network, iroha_client) = <Network>::start_test_with_runtime(4, Some(10_965));
     wait_for_genesis_committed(&network.clients(), 0);
     let pipeline_time = Configuration::pipeline_time();
 
+    let parameter = Parameter::new(
+        ParameterId {
+            name: Name::from_str("MaxTransactionsInBlock").expect("Failed to parse Name"),
```

Comment:
I used `expect` where function return `()`. Do you think we should change return value to `Result`?

Reply:
Here's what I think. In tests, and in most situations both the error and the context matter. `unwrap` and just `?` give us the error message, `expect` adds a single string as context. `eyre::Result` however, allows us to nest functions, and resturcture tests without much overhead. 

This means that returning `eyre::Result` is more compact and flexible. 
- [ ] $O(1)$ overhead of returning `Ok(())` and adding return type. 
- [ ] Can easily and compactly nest fallible operations with the equivalent of `unwrap`, assuming our error messages provide enough context. E.g. `.parse()?` and `to_value()?`. 
- [ ] Can provide more context with `wrap_err`. Unlike `expect` we can do this more than once, and in the test code. We don't really feel it now, because we don't have many property-based unit tests and formal verification.  

So functionally speaking `eyre::Result` and `unwrap()`, `expect()` approaches are similar, but where there's a difference `eyre` is superior with minor overhead. 

## Tests vs code

We should also touch upon the difference between tests and code. We follow a strict guideline on panics in the code. 

We only use `expect` in order to use the string as a comment on what invariant must be upheld for the panic not to take place. Ideally with formal verification, we should have zero panics beyond the init stage, where even unrecoverable errors traverse the call-stack to be provided with context. So in effect, all instances of `unwrap` and `expect` are replaceable with `unwrap_unchecked`, we just write a memo in `expect` to remind us why. 

Tests should be a natural extension of this logic. The errors must contain the necessary context when they fail, so `"alice @wonderland".parse()?` should give you enough information without any additives. We say `parse().expect("Valid")` which is equivalent to `parse().unwrap()`. 

We want to add context. Problem is, the context 
```rust
"alice@wonderland".parse().expect("Valid, because the string contains no whitespace, has a single '@' character and is not empty after")
```
is too much stuff for multiple `AccounId`s `parse()`d right after one another. We can then say `unwrap` is OK, but it's still 5 chars more than `?`, plus you'd need to disable it with `clippy` which a contributor might forget. 

Easier to just say: use `eyre::Result`.

Reply:
Ok, valid point and thanks for your opinion on this topic. I will change affected tests to use `eyre`.